pootle/static/js/editor/utils/strings.js

```diff
@@ -11,12 +11,12 @@ export const Formats = {
   PLURR: 'PLURR',
 };
 
-/**
- * Returns the string format detected for `value`.
- */
 export function detectFormat(value) {
   if (value.search(/{[^{}]*:.*?}/) !== -1) {
```

Comment:
I would suggest adding comments that say that the first case matches Plurr choice strings, e.g. `{FOO:xxxxx}`, and the other one matches simple placeholders like `{FOO}`.

Reply:
will do